src/main/flight/pid.h

```diff
@@ -76,13 +83,16 @@ typedef struct pidProfile_s {
 
 PG_DECLARE_PROFILE(pidProfile_t, pidProfile);
 
+static inline const pid8_t * pidBank() { return STATE(FIXED_WING) ? pidProfile()->bank_fw : pidProfile()->bank_mc; }
```

Comment:
You could define this as something like:
```
static inline const pid8_t * pidBank(int i) { return STATE(FIXED_WING) ? pidProfile()->bank_fw[i] : pidProfile()->bank_mc[i]; }
```
then you could use (eg):
```
pidBank(PID_ROLL)->I
```
Not 100% sure the syntax is right (haven't run it through the compiler), but you get the idea.
            
  Write